swift/apple/FirezoneNetworkExtension/Adapter.swift

```diff
@@ -276,38 +272,8 @@ class Adapter {
   }
 
   func setInternetResourceEnabled(_ enabled: Bool) {
-    workQueue.async { [weak self] in
```

Comment:
@jamilbk I also removed the work queue because all we are doing is setting a field on self and calling an FFI function that pushes a messages into a channel. Neither of these should be blocking of any form so no need to do it in a task I think.

Reply:
Restored.